### PR TITLE
[DO NOT MERGE] Add JA ver, Yahoo JAPAN case study

### PR DESCRIPTION
**Pull request by agektmr**
This pull request add the Japanese version of Yahoo JAPAN identity case study at https://web.dev/yahoo-japan-identity-case-study/

### CODE REVIEW

**Comment by agektmr** [comment]
Let us wait for an approval from Yahoo JAPAN.

**Comment by heyawhite** [comment]
@agektmr I'll approve for now, and add DO NOT MERGE to the title. Once they've approved, you can merge.

**Review by heyawhite — APPROVED** [review]

**Pull request merged**
